doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2224,3 +2224,13 @@ This is the local file system path to be used with the LocalSubmitToArchiveComma
 +++++++++++++++++++
 
 These are the bucket and project names to be used with the GoogleCloudSubmitToArchiveCommand class. Further information is in the :ref:`Google Cloud Configuration` section above.
+
```

Comment:
The below is missing the MicroProfile Config names. IMHO we should provide those, otherwise they are likely to stay unknown.

FWIW: there is an example for the database connections in the upper config section.

Reply:
I had to remove use of the microprofile name for the installationname - could not resolve some recursion problem. (trying to get to adding an issue). That said, my thought was that it probably isn't useful to advertise while there aren't many/all settings working this way (not sure it makes sense to add the complexity of supporting mixed ways of doing settings for just one setting.) I'm open if the group consensus is the other way though. (FWIW - the microprofile-alias file will essentially serve as a list of these as they're added - the microprofile docs you created could point to that to start for those who want to experiment/need to for k8s.)